src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Cmdlets/InstallPackageCommand.cs

```diff
@@ -99,6 +100,15 @@ protected override void ProcessRecordCore()
             {
                 await InstallPackageByIdAsync(Project, Id, ResolutionContext, this, WhatIf.IsPresent);
             }
+            catch (FatalProtocolException ex)
+            {
+                // Wrap FatalProtocolException coming from the server with a user friendly message
+                var error = String.Format(CultureInfo.CurrentUICulture, Strings.Exception_PackageNotFound, Id, Source);
+                Log(MessageLevel.Error, error);
+
+                // Additional information about the exception can be observed by using the -verbose switch with the install-package command
+                Log(MessageLevel.Debug, ExceptionUtilities.DisplayMessage(ex));
```

Comment:
looks a bit weird on a screenshot. The error message should come last. I'd change the order of log messages here.